browser/js/reviews/review-item/review-item.html

```diff
@@ -0,0 +1,14 @@
+<!-- <div class="col-xs-6"> -->
+		<div class="well text-center" style="height: 150px">Movie image</div>
+		<div class="panel-body">
+			<div class="list-group">
+				<div class="list-group-item"><strong>{{review.title}}, {{review.movie}} by {{review.user}}</strong></div>
+				<div class="list-group-item">Created: {{review.dateCreated}}</div>
+				<div class="list-group-item">Rating: {{review.rating}}<span class="glyphicon glyphicon-star"></span><span class="glyphicon glyphicon-star"></span><span class="glyphicon glyphicon-star"></span></div>
+				<div>{{review.content}}</div>
+				<div ng-repeat="bestFor in reviews.bestFor">{{bestFor}}</div>
```

Comment:
this will create lots of <divs>, might be better to ng-repeat on a <span> inside the div.